src/middleware.ts

```diff
@@ -1,18 +1,18 @@
 import { authMiddleware } from "@clerk/nextjs";
- 
+
 // See https://clerk.com/docs/references/nextjs/auth-middleware
 // for more information about configuring your Middleware
 export default authMiddleware({
   // Allow signed out users to access the specified routes:
-  publicRoutes: ['/'],
+  publicRoutes: ["/", "/api/dummy"],
```

Comment:
Non credo che in Clerk questo sia il modo corretto per autenticare un'api. Se togliamo questa public route, la chiamata da 401. Ho provato ad usare `apiRoutes` invece di mettere l'api come publica, ma non funge, dovremmo indagare meglio a riguardo

Reply:
Eh sai che non lo so neanche io come. Forse c'e' un modo tipo regex, che sotto usa proprio quella per re includere tutte le API